overlord/snapstate/autorefresh.go

```diff
@@ -739,8 +742,19 @@ var asyncPendingRefreshNotification = func(ctx context.Context, refreshInfo *use
 // maybeAsyncPendingRefreshNotification broadcasts desktop notification in a goroutine.
 //
 // The notification is sent only if no snap has the marker "snap-refresh-observe"
-// interface connected.
+// interface connected and the "refresh-app-awareness-ux" experimental flag is disabled.
 func maybeAsyncPendingRefreshNotification(ctx context.Context, st *state.State, refreshInfo *userclient.PendingSnapRefreshInfo) {
+	tr := config.NewTransaction(st)
+	experimentalRefreshAppAwarenessUX, err := features.Flag(tr, features.RefreshAppAwarenessUX)
+	if err != nil && !config.IsNoOption(err) {
+		logger.Noticef("Cannot send notification about pending refresh: %v", err)
```

Comment:
I don't believe the "WARNING:" prefix is needed (maybe even it needs to be removed in other places) since the warnings command already displays "warning:"
```bash
zeyad@z16:~$ snap warnings 
last-occurrence:  today at 09:59 EET
warning: |
  WARNING: Cannot send notification about pending refresh
---
last-occurrence:  today at 09:59 EET
warning: |
  Cannot send notification about pending refresh
```

Reply:
OTOH, the whole file doesn't do the prefix.